packages/client/src/Spacetime.ts

```diff
@@ -35,12 +37,13 @@ export class Spacetime {
 
   collection<T=any> (id: string): Collection<T> {
     if (this.collections[id]) return this.collections[id]
-    const c = new Collection<T>(id, this.client)
-    this.collections[id] = c
+    const path = this.config.defaultNamespace ? `${this.config.defaultNamespace}/${id}` : id
```

Comment:
There is no way to opt out of prepending the default namespace in `collection(id)` if your client has a defaultNamespace defined, so you can't read from collections in other namespaces, or the `$collections` collection. Probably not a big deal? Might be annoying for some users though if they have to create another client or use a workaround

Reply:
Good point, maybe we can use absolute and relative paths. So if you use `/` we wont' use the default path.